### ISSUE TITLE
[BUG/ISSUE] GCHP v 13.1.0 : CMake Error install PROGRAMS given no DESTINATION!

### ISSUE THREAD

**Issue by mtsivlidou** [geoschem/GCHP#118]
### Description of the problem
Hello, 
I am trying to compile GCHP v 13.1.0 and I get the following error: 
-- [MAPL] (2.6.3)
CMake Error at src/MAPL/Apps/CMakeLists.txt:11 (install):
  install PROGRAMS given no DESTINATION!

I would appreciate it if you could help me with this error. 
Thank you in advance, 
Maria

### GEOS-Chem version
GCHP v 13.1.0

### Description of code modifications
No modifications.

### Log files
- Build log (if applicable): 
[CMakeError.log](https://github.com/geoschem/GCHP/files/6853597/CMakeError.log)
[CMakeOutput.log](https://github.com/geoschem/GCHP/files/6853598/CMakeOutput.log)

### Software versions
- Compilers (Intel or GNU, and version):  gcc/9.3.0
- NetCDF version: netcdf/4.7.4-openmpi4.0.3.3-gfortran4.8.5
- MPI version: openmpi/gnu/4.0.3.3
- ESMF version: 8.0.1

- cmake/3.13.0
- git/2.9.5
- hdf5/1.10.2-openmpi

**Comment by LiamBindle** [comment]
Hi Maria, thanks for pointing this out. I'm looking into it.

In the meantime you can copy the built executable from `bin/gchp` (in your build directory) to your run directory.

**Comment by mtsivlidou** [comment]
Hello, 
The executable has not been created. The GCHP/build/bin folder is empty. 
The last line of the compilation are: 

-- Performing Test FORTRAN_COMPILER_SUPPORTS_ASSUMED_TYPE
-- Performing Test FORTRAN_COMPILER_SUPPORTS_ASSUMED_TYPE: SUCCESS
-- Performing Test FORTRAN_COMPILER_SUPPORTS_FINDLOC
-- Performing Test FORTRAN_COMPILER_SUPPORTS_FINDLOC: SUCCESS
-- Found MPI: TRUE (found version "3.1")  
-- BASEDIR: IGNORE
-- HDF5: Using hdf5 compiler wrapper to determine C configuration
-- Found NetCDF4: 1  
-- Found NetCDF4: 1  
-- Found MPI: TRUE (found version "3.1") found components:  C CXX Fortran 
-- ---------------------------------------------------------
-- [gchp_ctm] (13.1.0) [106b8f7]
-- Performing Test _LOGICAL_DEFAULT_KIND: SUCCESS (value=4)
-- Performing Test _INT_DEFAULT_KIND: SUCCESS (value=4)
-- Performing Test _ISO_INT8: SUCCESS (value=1)
-- Performing Test _ISO_INT16: SUCCESS (value=2)
-- Performing Test _ISO_INT32: SUCCESS (value=4)
-- Performing Test _ISO_INT64: SUCCESS (value=8)
-- Performing Test _REAL_DEFAULT_KIND: SUCCESS (value=4)
-- Performing Test _DOUBLE_DEFAULT_KIND: SUCCESS (value=8)
-- Performing Test _ISO_REAL32: SUCCESS (value=4)
-- Performing Test _ISO_REAL64: SUCCESS (value=8)
-- Performing Test _ISO_REAL128: SUCCESS (value=16)
-- Using submodule /tmpdir/tsivlido/v13.1/GCHP/src/gFTL-shared/extern/gFTL
-- Performing Test SUPPORT_FOR_ASSUMED_TYPE
-- Performing Test SUPPORT_FOR_ASSUMED_TYPE: SUCCESSS
-- Performing Test SUPPORT_FOR_C_LOC_ASSUMED_SIZE
-- Performing Test SUPPORT_FOR_C_LOC_ASSUMED_SIZE: FAILURE
-- Performing Test SUPPORT_FOR_MPI_ALLOC_MEM_CPTR
-- Performing Test SUPPORT_FOR_MPI_ALLOC_MEM_CPTR: FAILURE
-- ---------------------------------------------------------
-- [MAPL] (2.6.3)
CMake Error at src/MAPL/Apps/CMakeLists.txt:11 (install):
  install PROGRAMS given no DESTINATION!


-- Building FV dynamics as hydrostatic
=================================================================
HEMCO 3.0.0
Current status: 3.0.0
=================================================================
=================================================================
GEOS-Chem 13.1.0 (science codebase)
Current status: 13.1.2
=================================================================
-- Settings:
  * OMP:          ON  OFF
  * USE_REAL8:    ON  OFF
  * APM:          ON  OFF
  * RRTMG:        ON  OFF
  * GTMM:         ON  OFF
  * LUO_WETDEP:   ON  OFF
-- Configuring incomplete, errors occurred!
See also "/tmpdir/tsivlido/v13.1/GCHP/build/CMakeFiles/CMakeOutput.log".
See also "/tmpdir/tsivlido/v13.1/GCHP/build/CMakeFiles/CMakeError.log".

**Comment by LiamBindle** [comment]
Thanks for the clarification. I tracked the error down. The problem is `src/MAPL/Apps/CMakeLists.txt` is using an argument that wasn't introduced until CMake 3.14.

You can fix it with the following modification to `src/MAPL/Apps/CMakeLists.txt`:
```diff
 install (PROGRAMS
   MAPL_GridCompSpecs_ACG.py
   mapl_acg.pl
   mapl_stub.pl
-  ${INSTALL_ARG_TYPE_SYSCONF}
+  DESTINATION etc
   )
```

@mtsivlidou does that fix the error for you?

**Comment by LiamBindle** [comment]
The fix is implemented in this PR: https://github.com/geoschem/MAPL/pull/11.

I'll merge it into `dev` once it's reviewed.

**Comment by mtsivlidou** [comment]
My src/MAPL/Apps/CMakeLists.txt initially was:
install (PROGRAMS
  MAPL_GridCompSpecs_ACG.py
  mapl_acg.pl
  mapl_stub.pl
  TYPE SYSCONF
  )
~    

After your suggestion the error is fixed. Thank you!

**Comment by LiamBindle** [comment]
No problem! Sorry about that!

I'll leave this open until geoschem/MAPL#11 is merged.

**Comment by LiamBindle** [comment]
Fixed in 9b2e21c. Will be released in 13.1.3.

Thanks for reporting this @mtsivlidou!
